source/materials/MaterialsList.cc

```diff
@@ -187,13 +187,18 @@ namespace materials {
     G4Material* mat = G4Material::GetMaterial(name, false);
 
     if (mat == 0) {
-      G4NistManager* nist = G4NistManager::Instance();
 
       mat = new G4Material(name, ArgonDensity(pressure), 1,
         kStateGas, temperature, pressure);
 
-      G4Element* Ar = nist->FindOrBuildElement("Ar");
+      G4Element* Ar = new G4Element(name, "Ar", 1);
```

Comment:
Why do you think we need to change that? Aren't we using natural argon? Or is Geant4-defined argon wrong?

Reply:
I was going to include all the isotopes of ar, but their natural percentages are too low. Reverted back to just Ar 40 through G4.